public/app/features/explore/Explore.tsx

```diff
@@ -364,19 +363,13 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
                       )}
                       {!showStartPage && (
                         <>
-                          {showMetrics && (
-                            <ExploreGraphPanel
-                              ariaLabel={selectors.pages.Explore.General.graph}
-                              series={graphResult}
+                          {showMetrics && graphResult && (
```

Comment:
Shouldn't showMetrics be true only in case graphResult is true?

Reply:
this check is only to let TypeScript happy.